templates/postgresql/exporter.yaml

```diff
@@ -79,11 +79,17 @@ spec:
               drop:
                 - ALL
 
-    {{- if .Values.global.nodeSelector }}
+    {{- if .Values.database.nodeSelector }}
```

Comment:
nit: should we call it postgresqlexporter (or something else) instead of database?